ocsp/responder.go

```diff
@@ -85,25 +85,44 @@ type Responder struct {
 // from an OCSP request to an OCSP response is done by the Source;
 // the Responder simply decodes the request, and passes back whatever
 // response is provided by the source.
+// Note: The caller must use http.StripPrefix to strip any path components
```

Comment:
So nginx is doing OCSP GET? I will dig a little more, any help will be appreciated.